docs/changehistory/NextVersion.md

```diff
@@ -52,3 +52,11 @@ The following previously `alpha` or `beta` APIs have been promoted to `public`.
 ## Popout Widgets
 
 IModelApps, that use AppUi version "2", can now specify if a Widget can support being "popped-out" to a child popup window. The child window runs in the same javascript context as the parent application window. See [Child Window Manager]($docs/learning/ui/framework/ChildWindows.md) for more details.
+
+## Breaking API changes
+
+### [@bentley/imodeljs-backend](https://www.itwinjs.org/reference/imodeljs-backend/)
+
+To make it easier to use async APIs while exporting a schema, [IModelExportHandler.onExportSchema]($backend) has been made async and must return a promise.  For example, serialization APIs can be async, and previously to have custom async schema serialization, one would have to manually synchronize around their call to [IModelExportHandler.exportSchemas]($backend).
+
+[IModelTransformer.shouldExportSchema]($backend) now gets a [SchemaKey]($ecschema-metadata) schema key as argument, instead of a full [Schema]($ecschema-metadata). This puts the burden on the caller for loading a full schema if they need it, but makes most use cases for `shouldExportSchema` checks more performant.
```

Comment:
I would rather encourage a quick answer in `shouldExportSchema` (don't load the schema) and then skip over it in `onExportSchema` if you need it to be loaded to get your answer.